staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1704,6 +1704,7 @@ type RemoveVolumeOptions struct {
 // KubeVirtConfiguration holds all kubevirt configurations
 // +k8s:openapi-gen=true
 type KubeVirtConfiguration struct {
+	CPUArch                string                  `json:"-"`
```

Comment:
This field can be kept, but it should be set at initialization when the whole default config is constructed. It should not later on be used to determine how to set the defaults.